client/modules/core/main.js

```diff
@@ -158,6 +159,46 @@ export default {
     return reactionState;
   },
 
+  /**
+   * absoluteUrl
+   * @summary a wrapper method for Meteor.absoluteUrl which sets the rootUrl to
+   * the current URL (instead of defaulting to ROOT_URL)
+   * @param {String} [path] A path to append to the root URL. Do not include a leading "`/`".
+   * @param {Object} [options]
+   * @param {Boolean} options.secure Create an HTTPS URL.
+   * @param {Boolean} options.replaceLocalhost Replace localhost with 127.0.0.1. Useful for services that don't recognize localhost as a domain name.
+   * @param {String} options.rootUrl Override the default ROOT_URL from the server environment. For example: "`http://foo.example.com`"
+   * @return {String} URL for the given path and options
+   */
+  absoluteUrl(path, options) {
+    const o = _.extend({}, options);
```

Comment:
Can use `Object.assign({}, options)` instead of lodash `extend` here to clone as it doesn't look like there are any deep properties being mutated.

Also would prefer a more explicit variable name such as opts.